plutus-contract/src/Plutus/Contract/StateMachine.hs

```diff
@@ -252,7 +253,7 @@ waitForUpdateUntilTime ::
     -> POSIXTime
     -> Contract w schema e (WaitingResult state)
 waitForUpdateUntilTime sm timeoutTime = do
-    waitForUpdateUntilSlot sm $ TimeSlot.posixTimeToSlot timeoutTime
+    waitForUpdateUntilSlot sm $ TimeSlot.posixTimeToSlot def timeoutTime
```

Comment:
It feels like we should have access to the `SlotConfig` in `Contract`, maybe?

Reply:
In a new PR?

Reply:
Yes sure, just observing!